src/main/java/frc/robot/subsystems/drive/Drive.java

```diff
@@ -599,10 +601,11 @@ public boolean isDriveLiningUp() {
    * @return true if location is reef; false otherwise (processor / coral station)
    */
   public boolean isDesiredLocationReef() {
-    return !(desiredLocation == DesiredLocation.CoralStationLeft
-            || desiredLocation == DesiredLocation.CoralStationRight
-            || desiredLocation == DesiredLocation.Processor)
-        && !goToIntake; // only want reef if intake is false
+    return isLocationAlgaeIntake(desiredLocation) && goToIntake
```

Comment:
Can you add parentheses around this algae condition?

Even better can we split this huge boolean expression into 2: isCoralReefTarget, isAlgaeReefTarget and OR them together 

That would be way more readable